### PR TITLE
Mambaforge -> Miniforge3

### PR DESCRIPTION
**Pull request by maxnoe**
Miniforge3 now includes mamba by default, is identical to Mambaforge and Mambaforge is deprecated, see:https://github.com/conda-forge/miniforge#miniforge-pypy3

### CODE REVIEW

File: install/windows-10.md [hunk 1]
```diff
@@ -343,12 +343,12 @@ mkdir -p ~/.local
 Dieser Befehl erstellt einen Ordner mit dem Namen `.local`, falls dieser nicht schon existiert
 und tut gar nichts, falls dieser Ordner schon existiert.
 
-### <a id="Mambaforge"></a>Mambaforge: Python Installation
+### <a id="Miniforge3"></a>Miniforge3: Python Installation
```
Comment (aknierim) [inline install/windows-10.md:346]: We should keep this consistent: Either "Miniforge3: Python Installation" (as in Win10/11) or "Python Installation: Miniforge3" (as in Linux/MacOS)

File: install/windows-11.md [hunk 2]
```diff
@@ -298,12 +298,12 @@ mkdir -p ~/.local
 Dieser Befehl erstellt einen Ordner mit dem Namen `.local`, falls dieser nicht schon existiert
 und tut gar nichts, falls dieser Ordner schon existiert.
 
-### <a id="Mambaforge"></a>Mambaforge: Python Installation
+### <a id="Miniforge3"></a>Miniforge3: Python Installation
```
Comment (aknierim) [inline install/windows-11.md:301]: See previous comment

**Review by aknierim — CHANGES_REQUESTED** [review]

File: install/windows-10.md (hunk 1 above)
Comment (maxnoe) [reply install/windows-10.md:346]: I did a simple search and replace. So if it isn't consistent now, it wasn't before...

Comment (aknierim) [reply install/windows-10.md:346]: Yeah, I just thought we could kill two birds with one stone while we're at it. Other than that it's perfectly fine imo

Comment (jpherdi) [reply install/windows-10.md]: done

**Review by aknierim — APPROVED** [review]

**Pull request merged**